### PR TITLE
Feature/molecule pagination

### PR DESCRIPTION
**Pull request by juanmaguitar**
Implementation of the `MoleculePagination` component
Demo  → https://sui-components-feature-molecule-pagination.now.sh/workbench/molecule/pagination/demo
JIRA → https://jira.scmspain.com/browse/SUIC-61
Paper → https://paper.dropbox.com/doc/ZbJqOKkqvQnknwbBWyDOw

### CODE REVIEW

**Review by devictoribero — COMMENTED** [review]
I think it looks really good! :eyes: 

IMHO it feels weird as UX that when we click the lowest or highest Pagination Tab, the prev or next Tab disappears. It makes the UI of the Pagination move.. And personally it's something that bothers me in all the webapps or pages.

Apart of this, really good job! :+1: 

I liked it a lot :fire: :fire:

File: components/molecule/pagination/src/index.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    return (
+      <div className={BASE_CLASS}>
+        <MoleculeButtonGroup>
+          {prevPage && (
+            <AtomButtom onClick={this.handleClickPrev}>
+              {PrevButtonIcon && (
+                <span className={CLASS_PREV_BUTTON_ICON}>
+                  <PrevButtonIcon />
+                </span>
+              )}{' '}
```
[the remark this answers is not in the document]
Comment (juanmaguitar) [reply components/molecule/pagination/src/index.js]: Autogenerated. I'll remove it

[the remark this answers is not in the document]
Comment (kikoruiz) [reply components/molecule/pagination/src/index.js]: I think it's fine: applied by IDE if you have Prettier installed and "fix on save" feature enabled.

**Review by kikoruiz — DISMISSED** [review]
Good job! 😃

**Review by devictoribero — DISMISSED** [review]
Amazingg!! Good job 😄

**Comment by JoanClaret** [comment]
Some observations:

- The buttons used in the paper pagination definition does not fit with any `atom/button` as i can see in the buttons paper specification
- This component does not fit with UX specification in terms of design
- This component is not responsive

Some of this comments are not strictly related with the PR, as they concerns the UX specificacion. 
Maybe somebody can bring some light?
Thank you!

**Comment by juanmaguitar** [comment]
I have added to the demo a new use case w/ a responsive behaviour w/ the help of the `LayoutMediaQuery` component. I have also used SVG arrows to match more the paper design.
https://sui-components-feature-molecule-pagination.now.sh/workbench/molecule/pagination/demo
In terms of design, this `MoleculePagination` component relies completely on `MoleculeButtonGroup` which relies  completely on `AtomButton`

**Review by devictoribero — APPROVED** [review]

**Review by kikoruiz — APPROVED** [review]

**Pull request merged**